Lib/test/test_tarfile.py

```diff
@@ -429,6 +429,11 @@ def test_premature_end_of_archive(self):
                 with self.assertRaisesRegex(tarfile.ReadError, "unexpected end of data"):
                     tar.extractfile(t).read()
 
+    def test_length_zero_header(self):
+        with self.assertRaisesRegex(tarfile.ReadError, "file could not be opened successfully"):
+            with tarfile.open(support.findfile('recursion.tar')) as tar:
+                tar.getmembers()
```

Comment:
I think this `tar.getmembers()` can be replaced with `pass` since it should never be called. Or you can call `tarfile.is_tarfile` instead of `with tarfile.open...`. Feel free to ignore this comment.

Reply:
Thank you for the comment, I replaced the `tar.getmembers()` with `pass` as you suggested